x/bridge/keeper/genesis_test.go

```diff
@@ -0,0 +1,14 @@
+package keeper_test
+
+// TestGenesis tests that the default genesis state is valid and vdlidates
+// that all genesis denoms appear in tokenfactory
+func (s *KeeperTestSuite) TestGenesis() {
+	// Get all tf denoms created by the bridge module
+	tfDenoms := s.GetBridgeTFDenoms()
```

Comment:
imo id rather go for the more verbose GetBridgeTokenFactoryDenoms, as it took me non negligible time to understand what it meant.